pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "pep-pre-commit-hooks"
+version = "0.1.0"
+description = "Public pre-commit hooks"
+authors = ["Lorenz Walthert <lorenz.walthert@icloud.com>"]
+readme = "README.md"
+packages = [
+    {include = "pep_pre_commit_hooks"}
+]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+
+[tool.poetry.scripts]
+verify-git-email = "pep_pre_commit_hooks.core:cli_verify_git_email"
+
+
+[tool.poetry.group.dev.dependencies]
+mypy = "^1.1.1"
+commitizen = "^2.42.1"
+black = "^23.1.0"
+ruff = "^0.0.254"
+pytest = "^7.2.2"
+pytest-cov = "^4.0.0"
+
+
```

Comment:
OK, so you want to run only the limited set of rules that are enabled by default?

https://github.com/charliermarsh/ruff#configuration

Reply:
I oped for the default. How did you select yours in https://github.com/Ponte-Energy-Partners/peprock/blob/main/pyproject.toml?

Reply:
Let me see if I c/p from our main repo and run ruff. We can always relax later. The other way round, refactors become bigger the longer we wait…

Reply:
I assume you went through all rules and removed the ones you felt were not helpful for us?

Reply:
I believe it's all rules that are currently available (see link above). Some rule are expected to be incompatible with each other, as per ruff docs.